connectors/sources/confluence.py

```diff
@@ -112,6 +145,36 @@ async def api_call(self, url):
                 )
                 await self._sleeps.sleep(RETRY_INTERVAL**retry_counter)
 
+    async def paginated_api_call(self, url_name, **url_kwargs):
```

Comment:
From a method called `paginate_...` I would expect to be able to set a page number and a page size to fetch. What does paginate mean in the context of this method?

Reply:
This method performs api calls to the client, by taking care of next page, if the next page url is returned by the previous call. 
It is not intended to be called multiple times for a given object. Instead once called, it internally handles the next pages calls.